packages/cli/src/__tests__/index-api.test.ts

```diff
@@ -71,7 +73,7 @@ afterEach(async () => {
   await tmpFolder.cleanup()
 })
 
-test('count', async () => {
+testIfV3ShouldPass('count', async () => {
```

Comment:
this file I really would have expected to pass.  Would like to understand what the issue is preventing it from passing today

Reply:
It makes me wonder if the changes I made to disable v3 functionality when the V4 env var is set are wrong somehow